contracts/FIFOMarket.sol

```diff
@@ -16,6 +16,11 @@ import "hardhat/console.sol"; // todo
 
 // todo emit events
 
+struct RoundRobinOrder {
```

Comment:
any reason you defined this outside the scope of the contract?

Reply:
Nope! Fixed